ironic_prometheus_exporter/app/exporter.py

```diff
@@ -0,0 +1,30 @@
+import configparser
+import logging
+import os
+
+from flask import abort, Flask
+app = Flask(__name__)
+LOG = logging.getLogger(__name__)
+
+
+@app.route('/metrics', methods=['GET'])
+def prometheus_metrics():
+    try:
+        config = configparser.ConfigParser()
+        config.read(os.environ.get('IRONIC_CONFIG'))
+        DIR = config['oslo_messaging_notifications']['location']
+    except Exception as e:
+        LOG.error(e)
+        abort(404)
```

Comment:
It's not necessary 404, any error will end up here. Although 404 is never correct for this endpoint, since there is no way users can provide something that cannot be found. It has to be 500, I guess.

Reply:
Ok o/